packages/loaders/src/utils/useCSSSVGAnimation.spec.tsx

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { render } from 'garden-test-utils';
+import { useCSSSVGAnimation } from './useCSSSVGAnimation';
+
+describe('useCSSSVGAnimation', () => {
+  beforeEach(() => {
+    document.elementFromPoint = jest.fn().mockReturnValue(document.createElement('svg'));
+  });
+
+  it('returns whether CSS transforms are supported', () => {
```

Comment:
With us mocking this utility in the other loader test I wanted to have something to ensure that it could run without throwing an error. 

Do you think it would be better to have an `expect(() => {}).no.toThrow()` test instead?

Reply:
> With us mocking this utility in the other loader test 

Maybe the mock in that test can be a complete black box and is assumed to work.

> Do you think it would be better to have an expect(() => {}).no.toThrow() test instead?

I think it would depend on the value gain of the test. This would not test that the functionality is working. What other benefits would this test provide since it doesn't test for correct behavior?

I'd support a `not.toThrow` test if you think it has value. I was more concerned about the test description `returns whether CSS transforms are supported` because the test didn't verify that.

Reply:
I've updated this test description and now use a `not.toThrow` assertion to make it more clear.